source/locker/contracts/Locker.sol

```diff
@@ -200,6 +197,10 @@ contract Locker is Ownable {
     address account,
     uint256 amount
   ) private {
+    require(
+      balances[account].add(amount) <= (type(uint256).max).div(100),
```

Comment:
It might be a bit more consistent to use `MAX_PERCENTAGE` here, instead of `100`.